packages/bz2.rb

```diff
@@ -49,6 +49,11 @@ def self.install
     system "ln -s libbz2.so.1.0.6 #{CREW_DEST_LIB_PREFIX}/libbz2.so.1.0"
     system "ln -s libbz2.so.1.0.6 #{CREW_DEST_LIB_PREFIX}/libbz2.so.1"
     system "ln -s libbz2.so.1.0.6 #{CREW_DEST_LIB_PREFIX}/libbz2.so"
+
+    # Move manpages
+    system "mkdir -p #{CREW_DEST_PREFIX}/share/man/"
+    system "cp -a #{CREW_DEST_PREFIX}/man/* #{CREW_DEST_PREFIX}/share/man/"
```

Comment:
This could be:
```
system "mv #{CREW_DEST_PREFIX}/man #{CREW_DEST_PREFIX}/share/"
```
Then the next line is not needed.

Reply:
Ah, thank you. Wasn't sure if `#{CREW_DEST_PREFIX}/share/man` would be empty